resources/i18n/en.json

```diff
@@ -340,7 +340,8 @@
                 "PROCESS_VARIABLES": "Process variables",
                 "EXPRESSION": "Condition Expression",
                 "FORM_SELECTOR": "Form selector",
-                "DEFAULT_SEQUENCE_FLOW": "Default sequence flow"
+                "DEFAULT_SEQUENCE_FLOW": "Default sequence flow",
```

Comment:
Why do we need to touch this file at all? This block supposed to be deleted completely. We should only modify the language file inside the module, I reckon.